composer.json

```diff
@@ -138,6 +138,7 @@
         "contao/manager-plugin": "<2.0 || >=3.0",
         "doctrine/persistence": "1.3.2",
         "symfony/config": "<4.4.2",
+        "symfony/mime": "4.4.* <4.4.10 || 5.0.* <5.0.10 || 5.1.0",
```

Comment:
Contao is not yet compatible with Symfony 5 and most likely we will never support Symfony 5.0 at all (most likely the requirement will be `"symfony/mime": "4.4.* || 5.1.*"`), therefore we should not add the conflict here IMHO.